on-chain/package.json

```diff
@@ -1,21 +1,20 @@
 {
   "dependencies": {
     "base64-sol": "github:brechtpd/base64",
-    "dotenv": "^16.4.7",
     "npm": "^10.9.0"
   },
   "name": "on-chain",
   "version": "1.0.0",
-  "description": "You should not do this again, but here are the steps documenting the process.",
   "main": "index.js",
   "devDependencies": {
     "@nomicfoundation/hardhat-toolbox": "^5.0.0",
+    "@openzeppelin/contracts": "^5.2.0",
```

Comment:
Are these dev dependencies?  I am still fuzzy on how npm + smart contracts differentiate dev vs prod dependencies, so what you have may be correct.  I'm really not sure.

Reply:
I ready about it seems that dev are the ones only used in test. I reorganized.